src/get-config-base.js

```diff
@@ -1,4 +1,30 @@
 module.exports = () => ({
   parser: require.resolve('babel-eslint'),
   plugins: ['yola'],
+  env: {
+    browser: true,
+  },
+  overrides: [
+    {
+      files: [
+        '**/*-spec.js',
```

Comment:
All these could be captured with: `**/*spec.js*`

Reply:
True, but I'm afraid of such wide capturing 👻.